build.assets/Dockerfile

```diff
@@ -16,8 +16,9 @@ RUN apt-get install -q -y libc6-dev-i386
 RUN apt-get install -q -y net-tools
 RUN apt-get install -q -y tree
 
-RUN (groupadd jenkins --gid=$GID -o && useradd jenkins --uid=$UID --gid=$GID --create-home --shell=/bin/sh ;\
-     mkdir -p /var/lib/teleport && chown -R jenkins /var/lib/teleport)
+RUN (getent group  $GID || groupadd builder --gid=$GID -o; \
```

Comment:
This implementation is slightly different to what the old logic does as it will push ahead and try to add the user regardless of whether the group gets added or not, whereas the previous logic would stop if group adding fails. I don't think it makes much difference, though.

Reply:
Changed `;` to `&&` in 312fb5c.